.gitignore

```diff
@@ -1,23 +1,158 @@
+# editor/OS cache files
 .idea/
 .DS_Store
-__pycache__/
 .vscode/
+**/.DS_Store
 
 # jupyter notebooks
 .ipynb_checkpoints/
 
-# packaging
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
 dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
 *.egg-info/
-build/
+.installed.cfg
+*.egg
+MANIFEST
 
-# Coverage
+# Unit test / coverage reports
+htmlcov/
+.tox/
 .coverage
+.coverage.*
+.cache
+nosetests.xml
 coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+.coverage*
+*.py,cover
+.hypothesis/**
+.pytest_cache/**
 
 # Misc
 results/
 image_files*
 
 # datasets
 cifar*
+
+cleanlab/*.ipynb
```

Comment:
this is not valid in this repo, please remove this

Reply:
sure